src/rules/max-line-length/__tests__/index.js

```diff
@@ -126,3 +126,32 @@ testRule(rule, {
     column: 36,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [ 20, { ignore: "comments" } ],
+
+  accept: [ {
+    code: "/* comment that is too long */",
+  }, {
+    code: "       /* comment that is too long */",
+  }, {
+    code: "/* short */ a { color: 0; }",
+  }, {
+    code: "a {}\n/* comment that is too long\n*/ a { color: 0; top: 0; }",
```

Comment:
I'm not sure this should be allowed. Or it should be a different option. Because this is essentially different than a line that is only a comment: the result of this long comment is that *code* is being pushed to the right, which might leave your editor viewport.